pyccel/parser/syntactic.py

```diff
@@ -50,6 +50,7 @@
 from pyccel.ast.core import _atomic
 from pyccel.ast.core import create_variable
 
+
```

Comment:
Please checkout changes so wee don't see a file which wasn't changed in the review

Reply:
This can be done with the command
`git checkout master -- pyccel/parser/syntactic.py`